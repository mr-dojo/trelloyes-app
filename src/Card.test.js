import React from 'react';
import Card from './Card';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';


describe('Card componenet', () => {
  it ('Renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Card title='Test' content='Content Sample Paragraph' />, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  it ('Renders the UI with no changes', () => {
    const tree = renderer
      .create(<Card title='Test' content='Content Sample Paragraph'/>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});