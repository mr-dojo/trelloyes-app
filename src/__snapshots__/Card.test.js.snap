// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card componenet Renders the UI with no changes 1`] = `
<div
  className="Card"
>
  <button
    type="button"
  >
    Delete
  </button>
  <h3>
    Test
  </h3>
  <p>
    Content Sample Paragraph
  </p>
</div>
`;

exports[`Renders the UI with no changes 1`] = `
<div
  className="Card"
>
  <button
    type="button"
  >
    Delete
  </button>
  <h3>
    Test
  </h3>
  <p>
    Content Sample Paragraph
  </p>
</div>
`;
